mtc-wasm/assets/js/js_ffi.js

```
@@ -0,0 +1,512 @@
+            function onScanFailure(errorMessage) {
+                throw errorMessage;
+            }
+
+            activeHtml5QrcodeScanner = new Html5Qrcode(elementId);
+            await activeHtml5QrcodeScanner.start(camera_id, config, onScanSuccess, onScanFailure);
+            return toggleHtml5QrcodeScannerTorch(torch_on);
+        } catch (error) {
+            console.error(error);
+            reject(error);
+        }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)